Fabric.Authorization.AccessControl/src/app/modules/access-control/member-list/member-list.component.ts

```diff
@@ -1,9 +1,11 @@
 import { Component, OnInit } from '@angular/core';
 import { Observable } from 'rxjs/Rx';
 
+import { AccessControlConfigService } from '../../../services/access-control-config.service';
+
 import { FabricAuthMemberSearchService } from '../../../services/fabric-auth-member-search.service';
-import { AuthMemberSearchRequest } from '../../../models/authMemberSearchRequest';
-import { AuthMemberSearchResult } from '../../../models/authMemberSearchResult';
```

Comment:
Minor note: the `index.ts` file in the `models` (and other) directories allow `import` statements to be consolidated on a single line. The 2 lines below could become:

`import { AuthMemberSearchRequest, AuthMemberSearchResult } from '../../../models'`

Reply:
ok cool. i can switch that up.